contracts/AllDay.cdc

```diff
@@ -483,7 +483,6 @@ pub contract AllDay: NonFungibleToken {
                 AllDay.setByID.containsKey(setID): "setID does not exist"
                 AllDay.playByID.containsKey(playID): "playID does not exist"
                 SeriesData(id: seriesID).active == true: "cannot create an Edition with a closed Series"
-                SetData(id: setID).setPlayExistsInEdition(playID: playID) != true: "set play combination already exists in an edition"
```

Comment:
We want to loosen this restriction as long as the new edition with the same set/play is in a different tier.

Reply:
How are you enforcing that the same set/play is in a different tier?

Reply:
It is not enforced in the contract anymore, but I've added pre-condition check in the [create_edition](https://github.com/dapperlabs/nfl-smart-contracts/pull/21/files#diff-e4d7f97f5636b4be3a76ba07088ff51fc27ebbf9e4764df7ae94a917ce35c0fd) script that is used to mint new editions.